pages/celebrity-details.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+    <link rel="stylesheet" href="../css/custom.css">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400i,600,700,900" rel="stylesheet">
+  </head>
+  <body>
+    <div id=“fb-root”></div>
+      <script>(function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = “//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.10";
+        fjs.parentNode.insertBefore(js, fjs);
+      }(document, ‘script’, ‘facebook-jssdk’));</script>
+    <header>
+    <!-- Navbar -->
+      <nav class="navbar navbar-toggleable-md navbar-light bg-faded navbar-custom">

```

Comment:
This is creating two different versions of the header.

## Version 1

![image](https://user-images.githubusercontent.com/26730388/28977561-66698632-78f7-11e7-8f08-86bd5b3e2872.png)

## Version 2

![image](https://user-images.githubusercontent.com/26730388/28977600-8bdd66b8-78f7-11e7-97c9-4856a6a20aac.png)

As we discussed earlier, we need to not have any breakpoints in the design. We need version 2 only.